doc/install/install_via_centos7.md

```diff
@@ -527,6 +527,9 @@ curl http://${IP_ADDR}:8000/v2/webhooks
 # Troubleshooting
 
 ## Kamailio
+as of 2018-08-17, the blow would not work   but as a work around to force a working version:
+yum install librabbitmq-0.5.2-1.el7
+
```

Comment:
Weird, this was fixed, seems to have gotten excluded from the latest kazoo-kamailio, i am going to fix this. Once I fix there will be no need for a doc update, this is not intended behavior. I am fixing in the latest version and ensuring that going forward this is in the next versions generated. 

[root@apps001 ~]# repoquery -R kazoo-kamailio-0:4.2-27.el7.centos.noarch
kamailio = 5.0.4j-3.1
kamailio-db-kazoo = 5.0.4j-3.1
kamailio-http-utils = 5.0.4j-3.1
kamailio-kazoo = 5.0.4j-3.1
kamailio-outbound = 5.0.4j-3.1
kamailio-presence = 5.0.4j-3.1
kamailio-tls = 5.0.4j-3.1
kamailio-uuid = 5.0.4j-3.1
kamailio-websocket = 5.0.4j-3.1
kazoo-configs-kamailio = 4.2-20.el7.centos
librabbitmq = 0.5.2-1.el7
sudo
[root@apps001 ~]# repoquery -R kazoo-kamailio-0:4.2-28.el7.centos.noarch
kamailio = 5.0.4j-3.1
kamailio-db-kazoo = 5.0.4j-3.1
kamailio-http-utils = 5.0.4j-3.1
kamailio-kazoo = 5.0.4j-3.1
kamailio-outbound = 5.0.4j-3.1
kamailio-presence = 5.0.4j-3.1
kamailio-tls = 5.0.4j-3.1
kamailio-uuid = 5.0.4j-3.1
kamailio-websocket = 5.0.4j-3.1
kazoo-configs-kamailio = 4.2-21.el7.centos
sudo

Reply:
Should be fixed now... Since this only impacted 4.2-28 and that build has been fixed, we can probably Just close this pr. 

[root@apps001 ~]# repoquery -R kazoo-kamailio-0:4.2-28.el7.centos.noarch
kamailio = 5.0.4j-3.1
kamailio-db-kazoo = 5.0.4j-3.1
kamailio-http-utils = 5.0.4j-3.1
kamailio-kazoo = 5.0.4j-3.1
kamailio-outbound = 5.0.4j-3.1
kamailio-presence = 5.0.4j-3.1
kamailio-tls = 5.0.4j-3.1
kamailio-uuid = 5.0.4j-3.1
kamailio-websocket = 5.0.4j-3.1
kazoo-configs-kamailio = 4.2-21.el7.centos
librabbitmq = 0.5.2-1.el7
sudo